### ISSUE TITLE
Checking in the Pods Folder: Inconsistent Advice

### ISSUE THREAD

**Issue by wjk** [CocoaPods/guides.cocoapods.org#15]
I was studying the documentation for [using CocoaPods](http://guides.cocoapods.org/using/using-cocoapods.html), and I noticed an annoying inconsistency in [the section about adding the Pods directory to one’s `.gitignore`](http://guides.cocoapods.org/using/using-cocoapods.html#should-i-ignore-the-pods-directory-in-source-control).

The contents of the “Pros” and “Cons” sections seem to imply that one should not add the Pods folder to source control (aka put it in your `.gitignore`), but the text immediately above says “We recommend against adding the Pods folder to your `.gitignore`.” This (to me) implies the exact opposite of what the text below states. It seems to me that this is a simple grammatical oversight. If you want me to provide a PR for this, I will, but it is such a simple change I thought a committer handling it directly would be faster.

tl;dr: Remove the struck-through word: “We recommend ~~against~~ adding the Pods directory to your `.gitignore`.”
